ext/glm/CMakeLists.txt

```diff
@@ -1,19 +1,22 @@
 
+option(GLM_QUIET "No CMake Message" ON)
+option(GLM_TEST_ENABLE "Build unit tests" OFF)
 add_subdirectory(glm)
 
 add_library(inviwo::glm ALIAS glm)
 target_compile_definitions(glm INTERFACE
-    GLM_FORCE_RADIANS
     GLM_FORCE_SIZE_T_LENGTH
-    GLM_FORCE_UNRESTRICTED_GENTYPE
     GLM_ENABLE_EXPERIMENTAL
+    GLM_FORCE_UNRESTRICTED_GENTYPE
+    GLM_FORCE_SILENT_WARNINGS
 )
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
     # Needed when using clang since GLM fails to detect c++11/14 support
     # https://github.com/g-truc/glm/issues/620
-    target_compile_definitions(glm INTERFACE GLM_FORCE_CXX14)
+    #target_compile_definitions(glm INTERFACE GLM_FORCE_CXX14)
 endif()
 
-set_target_properties(glm_dummy PROPERTIES VERSION 0.9.9)
-ivw_folder(glm_dummy ext)
+#set_target_properties(glm_shared PROPERTIES VERSION ${GLM_VERSION})
```

Comment:
Decomment?

Reply:
I add it manually to get the patch version also...